data/events/2025/cluj-napoca/main.yml

```
@@ -0,0 +1,139 @@
+name: "2025-cluj-napoca" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2025" # The year of the event. Make sure it is in quotes.
+city: "Cluj-Napoca" # The displayed city name of the event. Capitalize it.
+event_twitter: "Devopsdays" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to City!" # Edit this to suit your preferences
+event_logo: "cluj-napoca.png" #
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+sharing_image: "TBD"
+masthead_background: "TBD"
```

```suggestion
sharing_image: ""
masthead_background: ""
```

If you don't have these images, just leave the field blank.

https://github.com/devopsdays/devopsdays-web/blob/main/themes/devopsdays-theme/reference.md